crates/core/tedge_mapper/src/c8y/fragments.rs

```diff
@@ -35,6 +36,32 @@ impl C8yAgentFragment {
         Ok(jsond)
     }
 }
+
+#[cfg(test)]
+pub fn get_tedge_version() -> Result<String, ConversionError> {
```

Comment:
Since only the very first line differs between the test and non-test versions of the `get_tedge_version` function,
I would rather keep a single version for that function and introduce a `get_tedge_version_command` function with two versions, for test and non-test.

Reply:
Updated to use the same function, but the assert_cmd crate was moved from dev-dependencies to dependencies